.ci/es-snapshots/Jenkinsfile_build_es

```diff
@@ -157,6 +157,9 @@ def buildArchives(destination) {
       ./gradlew -p distribution/archives assemble --parallel
```

Comment:
```suggestion
      ./gradlew -Dbuild.docker=true assemble --parallel
```

Reply:
@pugnascotia here are the archives which I see this produced from `distribution/docker`:

```
distribution/docker/docker-build-context/build/distributions/elasticsearch-8.0.0-SNAPSHOT-docker-build-context.tar.gz
distribution/docker/build/docker/elasticsearch-8.0.0-SNAPSHOT-linux-x86_64.tar.gz
distribution/docker/build/aarch64-docker/elasticsearch-8.0.0-SNAPSHOT-linux-aarch64.tar.gz
distribution/docker/build/aarch64-oss-docker/elasticsearch-oss-8.0.0-SNAPSHOT-linux-aarch64.tar.gz
distribution/docker/build/oss-docker/elasticsearch-oss-8.0.0-SNAPSHOT-linux-x86_64.tar.gz
distribution/docker/oss-docker-build-context/build/distributions/elasticsearch-oss-8.0.0-SNAPSHOT-docker-build-context.tar.gz
```

What it doesn't include are the `*-docker-image.tar.gz` archives which are [present on the release manager](https://snapshots.elastic.co/8.0.0-4c9cb790/summary-8.0.0-SNAPSHOT.html).

What is the suggested method for creating those assets?